src/core_modules/capture-core/components/FormFields/DateAndTime/D2Date/D2DateCalendar.component.js

```diff
@@ -140,6 +138,7 @@ class D2DateCalendar extends Component<Props> {
             <div
                 className={classes.container}
             >
+                { /* $FlowFixMe */ }
```

Comment:
After one of the flow js upgrades, flow has started complaining about "missing" props. These props aren't really missing, but rather passed on through our "passOnProps" object. I don't want to annotate props for every level where they are just passed on, but not used (a "interface" level where every prop is listed could make sense). Suppressing these errors for now, we should upgrade flow at some point and try to find a viable solution to this problem.